robot/rover/Arm/Arm.ino

```diff
@@ -1,894 +1,107 @@
+                    const uint8_t rawArgsLength) {
+  digitalWrite(LED_BUILTIN, !digitalRead(LED_BUILTIN));
+  // NOTE: this code is commented out since sending debug strings
+  // can overwhelm the bus if the teensy is seeing many invalid commands,
+  // which has happened in the past.
+
+  /*char buf[64];
+  snprintf(buf, sizeof(buf), "Invalid command id %d arg length %d", cmdID,
+           rawArgsLength);
+  internal_comms::Message* message =
+      commandCenter->createMessage(0, strlen(buf), buf);
+  commandCenter->sendMessage(*message);*/
 }
-/*! Each motor with limit switches needs to attach the limit switch and 2 ish interrupts.
-  this function also sets angle limits for the joints
- */
-void initLimitSwitches(void) {
-
-    // c for clockwise/counterclockwise, f for flexion/extension, g for gripper (assuming only one switch)
-    motor1.attachLimitSwitches(REVOLUTE_SWITCH, M1_LIMIT_SW_CW, M1_LIMIT_SW_CCW);
-    motor2.attachLimitSwitches(FLEXION_SWITCH, M2_LIMIT_SW_FLEX, M2_LIMIT_SW_EXTEND);
-    motor3.attachLimitSwitches(FLEXION_SWITCH, M3_LIMIT_SW_FLEX, M3_LIMIT_SW_EXTEND);
-    motor4.attachLimitSwitches(FLEXION_SWITCH, M4_LIMIT_SW_FLEX, M4_LIMIT_SW_EXTEND);
-    //motor5.attachLimitSwitches(REVOLUTE_SWITCH, M5_LIMIT_SW_CW, M5_LIMIT_SW_CCW);
-    motor6.attachLimitSwitches(FLEXION_SWITCH, M6_LIMIT_SW_FLEX, M6_LIMIT_SW_EXTEND);
 
-    attachInterrupt(motor1.limSwitchCw, m1CwISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor1.limSwitchCcw, m1CcwISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor2.limSwitchFlex, m2FlexISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor2.limSwitchExtend, m2ExtendISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor3.limSwitchFlex, m3FlexISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor3.limSwitchExtend, m3ExtendISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor4.limSwitchFlex, m4FlexISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor4.limSwitchExtend, m4ExtendISR, LIM_SWITCH_DIR);
-    //attachInterrupt(motor5.limSwitchCw, m5CwISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor6.limSwitchFlex, m6FlexISR, LIM_SWITCH_DIR);
-    attachInterrupt(motor6.limSwitchExtend, m6ExtendISR, LIM_SWITCH_DIR);
-
-    // set motor joint angle limits
-    motor1.setAngleLimits(M1_MIN_HARD_ANGLE, M1_MAX_HARD_ANGLE, M1_MIN_SOFT_ANGLE, M1_MAX_SOFT_ANGLE);
-    motor2.setAngleLimits(M2_MIN_HARD_ANGLE, M2_MAX_HARD_ANGLE, M2_MIN_SOFT_ANGLE, M2_MAX_SOFT_ANGLE);
-    motor3.setAngleLimits(M3_MIN_HARD_ANGLE, M3_MAX_HARD_ANGLE, M3_MIN_SOFT_ANGLE, M3_MAX_SOFT_ANGLE);
-    motor4.setAngleLimits(M4_MIN_HARD_ANGLE, M4_MAX_HARD_ANGLE, M4_MIN_SOFT_ANGLE, M4_MAX_SOFT_ANGLE);
-    // motor5.setAngleLimits(M5_MIN_HARD_ANGLE, M5_MAX_HARD_ANGLE, M5_MIN_SOFT_ANGLE, M5_MAX_SOFT_ANGLE); // this joint should be able to spin freely
-    motor6.setAngleLimits(M6_MIN_HARD_ANGLE, M6_MAX_HARD_ANGLE, M6_MIN_SOFT_ANGLE, M6_MAX_SOFT_ANGLE);
+void pong() {
+  digitalWrite(LED_BUILTIN, !digitalRead(LED_BUILTIN));
```

Comment:
This only toggles the LED, so it will stay on/off after the pong. This is undesirable, so better to blink it or not have visual feedback. Same for invalidCommand()